### PR TITLE
Add ability to loop over the ocp versions verifying if acceptable

### PR DESCRIPTION
**Pull request by shawn-hurley**
* Add variable for acceptable version `valid_ocp_versions`
* Overridable by the environment variable

### CODE REVIEW

File: roles/parse_operator_bundle/tasks/bundle_sanity_checks.yml [hunk 1]
```diff
@@ -1,9 +1,10 @@
 ---
-# Note: we probably want to make this more robust in the future to make a sliding scale that can be adjusted on the fly based on configuration?
 - name: "Verify ocp versions are valid"
   fail:
-    msg: "com.redhat.openshift.versions must only contain v4.5"
-  when: (ocp_versions|length==0) or (ocp_versions|length>1) or ocp_versions[0] != "v4.5"
+    msg: "com.redhat.openshift.versions can not contain {{ item }}"
```
Comment (scoheb) [inline roles/parse_operator_bundle/tasks/bundle_sanity_checks.yml:4]: should this say "The **label** com.redhat.openshift.versions can not contain {{ item }}"

**Comment by lslebodn** [comment]
@nmars we might need to add operator-bundle pipeline to the testsuite

**Comment by scoheb** [comment]
lgtm

**Comment by lslebodn** [comment]
Merging this MR would not be enough. We would need to fix CVP-1177.
Because current master broke bundle pipeline and fixing CVP-1178 would be good as well to prevent regression in operator bundle pipeline

**Comment by shawn-hurley** [comment]
@scoheb @dirgim rebased let me know if there is something else that you need me to change

File: roles/parse_operator_bundle/defaults/main.yml [hunk 2]
```diff
@@ -3,4 +3,8 @@ yq_bin_path: yq
 skopeo_bin_path: skopeo
 image_protocol: "docker://"
 operator_work_dir: /home/jenkins/agent/test-operator
-bundle_sanity_checks: true
\ No newline at end of file
+bundle_sanity_checks: true
+
+valid_ocp_versions:
+- "4.5"
```
Comment (dirgim) [inline roles/parse_operator_bundle/defaults/main.yml:9]: The expected OCP version format is "v4.x"

File: roles/parse_operator_bundle/defaults/main.yml [hunk 3]
```diff
@@ -3,4 +3,8 @@ yq_bin_path: yq
...
+- "4.6"
```
Comment (dirgim) [inline roles/parse_operator_bundle/defaults/main.yml:10]: Same as above, missing the "v" prefix

**Comment by yanirq** [comment]
@shawn-hurley  are there impediments preventing this PR to be merged ?

**Comment by lslebodn** [comment]
our CI is down atm

File: roles/parse_operator_bundle/defaults/main.yml [hunk 4]
```diff
@@ -8,4 +8,7 @@ required_labels:
   - 'operators.operatorframework.io.bundle.package.v1'
   - 'operators.operatorframework.io.bundle.channel.default.v1'
   - 'operators.operatorframework.io.bundle.channels.v1'
-  - 'com.redhat.openshift.versions'
\ No newline at end of file
+  - 'com.redhat.openshift.versions'
+valid_ocp_versions:
+- "v4.5"
+- "v4.6"
```
Comment (14rcole) [inline roles/parse_operator_bundle/defaults/main.yml:14]: nit: can you ident these two lines?

Comment (shawn-hurley) [reply roles/parse_operator_bundle/defaults/main.yml]: done

**Pull request merged**